Doc/requirements.txt

```diff
@@ -11,7 +11,7 @@ sphinx~=8.2.0
 
 blurb
 
-sphinxext-opengraph~=0.11.0
+sphinxext-opengraph~=0.12.0
```

Comment:
Does sphinxext-opengraph use SemVer?

If so, it's definitely used in production, time for 1.0.0 🚀

https://semver.org/#how-do-i-know-when-to-release-100